man/ctags.1.rst.in

```diff
@@ -1359,6 +1359,10 @@ are not listed here. They are experimental or for debugging purpose.
 	DESCRIPTION
 		Human readable description for the role.
 
+``--list-subparsers[=baselang|all]``
+	Lists the subparsers for a base language for either the specified
+	*baselang* or **all** languages, and then exits.
+
```

Comment:
Could you add something like

> **all** is used as default value if the option arugment is omitted.

?

Reply:
Maybe, we should add the same sentence also for other `--list-xxx[=language|all]` options.

Reply:
That's right. I will do.

Reply:
I'm preparing it in this PR.

Reply:
Oh, thank you.

Reply:
done